src/gen-use-table.py

```diff
@@ -338,6 +338,9 @@ def map_to_use(data):
 		if 0x1CF2 <= U <= 0x1CF3: UIPC = Right
 		if 0x1CF8 <= U <= 0x1CF9: UIPC = Top
 
+		if not (UIPC in [Not_Applicable, Visual_Order_Left] or USE in use_positions):
+			print ("%s %s %s %s %s" % (hex(U), UIPC, USE, UISC, UGC), file=sys.stderr)
+			continue
```

Comment:
I should figure out what is going on to know why this is failing, which of course it has some reason.

Reply:
```
0xf18 Bottom O Other Mn
0xf19 Bottom O Other Mn
0xf3e Right O Other Mc
0xf3f Left O Other Mc
0xf86 Top O Other Mn
0xf87 Top O Other Mn
```
It’s failing because Unicode assigns each of those characters an Indic_Positional_Category but not an Indic_Syllabic_Category.

Reply:
And any hint what should be here also? :)

Reply:
Set `UISC` for these characters after line 297.